sdk/python/feast/expediagroup/vectordb/vector_feature_view.py

```diff
@@ -50,6 +50,7 @@ class VectorFeatureView(BaseFeatureView):
 
     # inheriting from FeatureView wouldn't work due to issue with conflicting proto classes
     # therefore using composition instead
+    name: str
```

Comment:
feature_view already has an attribute `name`

Reply:
It did not reflect and threw an error when I tried to add name